### PR TITLE
Update docs from Alpha to Beta.

### PR DESCRIPTION
**Pull request by simitt**
Update to Beta and minor changes in example.

### CODE REVIEW

File: docs/configuration.asciidoc [hunk 1]
```diff
@@ -11,7 +11,7 @@ The recommended way to configure Elastic APM for Rails is to create a file `conf
 [source,yaml]
 ----
 ---
-server_url: 'http://localhost:8200'
+server_url: 'https://localhost:8200'
```
Comment (mikker) [inline docs/configuration.asciidoc]: You sure? We provide a self-signed cert for `localhost`?

Comment (simitt) [reply docs/configuration.asciidoc]: No we don't provide a cert. 
I wouldn't create an example with a `secret_token` and `http` though.

Comment (simitt) [reply docs/configuration.asciidoc]: @mikker I removed other changes than setting the `Beta` flag and leave it up to you if you want to change the example in the future.

Comment (mikker) [reply docs/configuration.asciidoc]: @simitt Thank you!

**Pull request merged**